app/assets/javascripts/discourse/app/components/topic-list/featured-topic.gjs

```diff
@@ -0,0 +1,43 @@
+import { on } from "@ember/modifier";
+import { htmlSafe } from "@ember/template";
+import TopicEntrance from "discourse/components/topic-list/topic-entrance";
+import TopicPostBadges from "discourse/components/topic-post-badges";
+import TopicStatus from "discourse/components/topic-status";
+import formatAge from "discourse/helpers/format-age";
+import { modKeysPressed } from "discourse/lib/utilities";
+
+const onTimestampClick = function (event) {
+  if (modKeysPressed(event).length) {
+    // Allow opening the link in a new tab/window
+    event.stopPropagation();
+  } else {
+    // Otherwise only display the TopicEntrance component
+    event.preventDefault();
+  }
+};
+
+const FeaturedTopic = <template>
+  <div data-topic-id={{@topic.id}} class="featured-topic --glimmer">
```

Comment:
Hmm, we don't want people to start relying on this, and then break it by removing later.

How about we add a new class to the old topic list, like `--legacy-raw-hbs`, and then use `not(.--legacy-raw-hbs)` where needed. That way it'll naturally be phased-out with the rest of the raw-hbs implementation